From 81e1be8d90ea6537d790549d11504d1992961e28 Mon Sep 17 00:00:00 2001
From: Sergey Fedorov <barracuda@macos-powerpc.org>
Date: Sat, 4 Jan 2025 22:54:54 +0800
Subject: [PATCH 1/2] Revert "temporarily disable the default S/DSCAL kernel"

This reverts commit 9b9c0aa5c9ab08df01fb5acfe83a50b4c7ce6bdd.
---
 kernel/power/KERNEL.PPC970 | 3 ---
 1 file changed, 3 deletions(-)

diff --git kernel/power/KERNEL.PPC970 kernel/power/KERNEL.PPC970
index d8e8c6646..fee5fa529 100644
--- kernel/power/KERNEL.PPC970
+++ kernel/power/KERNEL.PPC970
@@ -89,6 +89,3 @@ DROTKERNEL   = ../arm/rot.c
 CROTKERNEL   = ../arm/zrot.c
 ZROTKERNEL   = ../arm/zrot.c
 endif
-
-SSCALKERNEL = ../arm/scal.c
-DSCALKERNEL = ../arm/scal.c
-- 
2.47.1

